ACMILabs/works-embeddings
# #5 Update design from Tim/Lucie's feedback

Resolves #5

Let's do some more work on the design of the app.

### Acceptance Criteria
- [x] Update design from Tim's Figma file
- [x] After 5 seconds of inactivity make the works wiggle
- [x] After 30 seconds of inactivity refresh to the home page

### Relevant design files
* None

### Testing instructions
1. Start the app: `make up`
1. Note the new design takes into account [Lucie's feedback](https://trello.com/c/nJMQyN0U/185-works-explorer-work-on-ux-design-with-lucie-tim#comment-65ad97c0d8f4d20f940c0b68): http://localhost:8081
1. Set `REFRESH_TIMEOUT=32` in your `config.env`
1. `make up` and note that works wiggle after 5 seconds of inactivity, and reloads to the homepage after 30 seconds
